style.css

```diff
@@ -80,3 +83,172 @@ nav #myLogo {
   font-size: 18px;
   color: #6070ff;
 }
+
+.grid-box {
+  display: flex;
+  margin-left: auto;
+  margin-right: auto;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background: #e5e5e5;
+  border-radius: 8px;
+  width: 100%;
+  gap: 88px;
+  padding: 104px 0;
+}
```

Comment:
- [ ] [OPTIONAL] My colleague omitted this but please kindly consider using `css grid` on your main container for all project cards as included in the [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md). It will allow you to layout this efficiently now and later as you continue to make it responsive.